app/components/channel_list/categories/body/index.ts

```diff
@@ -99,12 +103,18 @@ const mapPrefName = (prefs: PreferenceModel[]) => of$(prefs.map((p) => p.name));
 
 const mapChannelIds = (channels: ChannelModel[]) => of$(channels.map((c) => c.id));
 
-type EnhanceProps = {category: CategoryModel; locale: string; currentUserId: string} & WithDatabaseArgs
+type EnhanceProps = {
+    category: CategoryModel;
+    locale: string;
+    currentUserId: string;
+    unreadsOnTop: boolean;
+    unreadChannelIds: Set<string>;
+} & WithDatabaseArgs
 
-const enhance = withObservables(['category'], ({category, locale, database, currentUserId}: EnhanceProps) => {
+const enhance = withObservables(['category', 'unreadChannelIds'], ({category, locale, database, currentUserId, unreadChannelIds}: EnhanceProps) => {
```

Comment:
Seems like changes in `locale` won't be re-triggering the queries. Is that intended? The same for `currentUserId`, but I guess that can be considered "constant".

Reply:
I think on `currentUserId` change - you're likely logging out and back in so that will be a complete change, probably not one we need to worry about here.

Interesting question about the `locale` though - how often do people change their language in the application, and then also expect the alphabetical sorting to change? I haven't tested this with different language sets yet - it's probably one for a different PR (i18n / internationalization?).

Reply:
The probability I guess it is low. I don't know for sure, but I think only a handful of languages change this order, and you would have to be changing between them. If you think it may be a problem to have it here, we can omit it, but I think it makes no harm adding the 'locale' to the list, and be more "correct".